.github/workflows/ci.yml

```diff
@@ -82,7 +82,6 @@ jobs:
       matrix:
         env:
           - TEST: pulp
-          - TEST: azure
```

Comment:
why removing azure tests?

Reply:
@fao89 I just ran the plugin template. I'm not sure why it did that

Reply:
I think your plugin-template is outdated

Reply:
looks like it was. Good catch